index.html

```diff
@@ -3,9 +3,147 @@
+                DesignX Company<br>
+                Web Design<br>
+                2023
+            </h1>
+            <div class="first-para">
+                <p>
+                Design agencies create innovative methods
+                of communication for clients and their brands
+                that create value for their businesses.
+                </p>
+            </div>
+            <h2 class="second-heading">
+                2023.07.3(MON) ~ 7(FRI)
+            </h2>
+            <p class="second-para">
+                @ Sadaf Tower, Tehran, Iran and more.
+            </p>
+        </section>
+    </div>
+<!--MAIN PROGRAM-->
+    <section id="program" class="main-program">
+        <div class="main-heading">
+            <h2>Main Program</h2>
+            <hr class="main-line">
+        </div>
+        <div class="program-cards">
+            <div class="cards">
+                <img src="images/icon1.png" alt="icon">
+                <h3>Keynote</h3>
+                <p>Renowned experts in the field deliver inspiring talks on the latest trends,
+                   technologies, and best practices in web design.
+                </p>
+            </div>
+            <div class="cards">
+                <img src="images/icon2.jpg" alt="icon">
+                <h3>Panel</h3>
+                <p>Panels bring together industry professionals to discuss
+                   specific topics or challenges related to web design. 
+                </p>
+            </div>
+            <div class="cards">
+                <img src="images/icon3.png" alt="icon">
+                <h3>Workshops</h3>
+                <p>Hands-on workshops allow participants to learn
+                   and practice specific skills or techniques. 
+                </p>
+            </div>
+            <div class="cards">
+                <img src="images/icon4.png" alt="icon">
+                <h3> Networking</h3>
+                <p>These sessions create opportunities for attendees to connect with peers
+                   , experts, and potential employers. 
+                </p>
+            </div>
+            <div class="cards">
+                <img src="images/icon5.png" alt="icon">
+                <h3>Competitions</h3>
+                <p>Some conferences may organize design competitions where participants can
+                   showcase their skills and creativity.
+                </p>
+            </div>
+            <div class="button">
+                <button type="submit">Join DesignX 2023</button>
+            </div>
+        </div>
+        <a class="link" href="/">See the Whole Program</a>
+    </section>
+<!--FEATURED-->
+    <section id="featured" class="featured-container">
+        <div class="featured-heading">
+            <h2>Featured Designers</h2>
+            <hr class="main-line">
+        </div>
+        <div id="featured-designers" class="featured-cards-container grid">
+            
+        </div>
```

Comment:
- You have done well so far, but please note that there is a little improvement to be made in this section.  Please add some more row spacing(horizontal spacing) between the speaker cards so it can match the design template a little more closely 👍 👍 
- I suggest that you also reduce the description font so it matches the design template, currently, the font looks a little too large. Please fix this 👍 👍 
Your project:
<img width="892" alt="erharha" src="https://github.com/78fatemanazari/Capstone-project/assets/73607512/36a9c7bd-d113-414b-97a7-d8deed81a14f">

Design template:
<img width="586" alt="rergr" src="https://github.com/78fatemanazari/Capstone-project/assets/73607512/2e4cef63-1eec-4523-8ce2-20680adbe17c">